src/components/10-atoms/input-file/index.scss

```diff
@@ -0,0 +1,172 @@
+.a-input-file {
```

Comment:
@LucaMele @NoahLiechti  Did you guys test this on internet explorer? (I mean the whole component, not the commented line, since you are using flexbox and stuff)

Reply:
pretty sure he did. @NoahLiechti ?

Reply:
@LucaMele Nein ich habe es auf Safari, Chrome und Firefox getestet, nicht aber auf IE (habe leider kein Browserstack). Könnte ich das an dich delegieren Luca?

Reply:
@NoahLiechti. Ah ok dann am Montag kommst du zu uns und wir testen es